static/src/javascripts/projects/common/modules/analytics/tech-feedback.js

```diff
@@ -0,0 +1,22 @@
+define([
+    'bean',
+    'fastdom',
+    'common/utils/$'
+], function (
+    bean,
+    fastdom,
+    $
+) {
+
+    return {
+        init: function () {
```

Comment:
ooh good spot, I'll do that.  Also it means the page width will be accurate even if they resized since they loaded it